web/src/beta/components/Blocks/ImageBlock/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import ImageBlock from ".";
+
+export default {
+  component: ImageBlock,
+} as Meta;
+
+type Story = StoryObj<typeof ImageBlock>;
+
+export const Default: Story = {
+  args: {
+    src: "http://placehold.it/150X150",
```

Comment:
we already have an public directory inside of .storybook with some sample photos. Please use those so we don't have to rely on an external API 🙏🏻